libkineto/include/GenericTraceActivity.h

```diff
@@ -87,6 +87,11 @@ class GenericTraceActivity : public TraceActivity {
   ActivityType activityType;
   std::string activityName;
 
+  int64_t seqNumber{-1};
```

Comment:
Adding four fields for this is a bit much. And it's very specific, both to PyTorch and to the way forward / backwards currently works.
At the very least this needs a big FIXME comment to explain this is a temporary and hacky implementation.

Reply:
Yes, it's a little heavy. I've considered moving these columns into "args" json string, but it will need parse the string in kineto side when using them to do correlation.
I've added FIXME to it. Please feel free to update it in a better way later on.

Reply:
I think most of the code does not belong in Kineto itself but in PyTorch Kineto profiler.

Reply:
OK. Moved to PyTorch side.